HHS/OPRE-OPS
# feat: :sparkles: adds AgreementChangesAccordion

## What changed

- adds: ✨AgreementChangesAccordion
- adds summary cards
- adds small summary card

## Issue

#1593 

## How to test

1. review an agreement
2. select budgetline actions
3. select budget lines
4. 🤩 see changes in AgreementChangesAccordion

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
